test/e2e/lib/mocha-hooks.js

```diff
@@ -64,6 +64,7 @@ afterEach( async function () {
 	let filenameCallback;
 
 	if ( this.currentTest.state === 'failed' ) {
+		console.error( this.currentTest.err );
```

Comment:
Reminder to remove this before merging.

Reply:
https://github.com/Automattic/wp-calypso/pull/45794/commits/9c359ce3ab8bad369c329b8fc44d43ff771fca83